sdk/src/asset_handlers/font_io.rs

```diff
@@ -212,6 +213,27 @@ pub(crate) fn checksum(bytes: &[u8]) -> Wrapping<u32> {
     chunks_cksum + frag_cksum
 }
 
+/// Computes a 32-bit big-endian OpenType-style checksum on the given byte
+/// array, which is presumed to start on a 4-byte boundary.
+///
+/// ### Parameters
+/// - `bytes` - Array of data to checksum
+/// - `bias`  - Starting byte offset.
+///
+/// ### Returns
+/// Wrapping<u32> with the data checksum. (Note that trailing pad bytes do not
+/// affect this checksum - it's not a real CRC.)
+#[allow(dead_code)]
+pub(crate) fn checksum_biased(bytes: &[u8], bias: u32) -> Wrapping<u32> {
+    match bias & 3 {
+        0 => checksum(bytes),
+        1 => Wrapping(BigEndian::read_u24(bytes)) + checksum(&(bytes[3..bytes.len()])),
+        2 => Wrapping(BigEndian::read_u16(bytes) as u32) + checksum(&(bytes[2..bytes.len()])),
+        3 => Wrapping(bytes[0] as u32) + checksum(&(bytes[1..bytes.len()])),
+        4_u32..=u32::MAX => todo!(),
```

Comment:
How.. would you get a 4 on a module 3, overflow?

Reply:
Right, I'm finally running into corners of Rust like this (and like the things `.chunks_exact` kicks back being runtime-sized).

Reply:
This kind of thing should be an open issue/request on the main language implementation (if not already).